egs/sprakbanken_swe/s5/local/sprak_data_prep.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# Copyright 2009-2012  Microsoft Corporation  Johns Hopkins University (Author: Daniel Povey)
+# Copyright 2013-2014  Mirsk Digital Aps (Author: Andreas Kirkedal)
+#Copyright 2016 KTH Royal Institute of Technology (Author: Emelie Kullmann)
+# Apache 2.0.
+
+
+dir=`pwd`/data/local/data
+lmdir=`pwd`/data/local/transcript_lm
+traindir=`pwd`/data/local/trainsrc
+testdir=`pwd`/data/local/testsrc
+rm -rf $lmdir $traindir $testdir $devdir
+mkdir -p $dir $lmdir $traindir $testdir $devdir
+local=`pwd`/local
+utils=`pwd`/utils
+
+
+. ./path.sh
+
+# Checks if python3 is available on the system and install python3 in userspace if not
+# This recipe currently relies on version 3 because python3 uses utf8 as internal 
+# string representation
+
+if ! which python3 >&/dev/null; then
+  echo "Installing python3 since not on your path."
+  pushd $KALDI_ROOT/tools || exit 1;
+  extras/install_python3.sh || exit 1;
+  popd
+fi
+
+if [ ! -d $dir/download ]; then
+    mkdir -p $dir/download/0467-1 $dir/download/0467-2 $dir/download/0467-3
+fi 
+
+echo "Downloading and unpacking sprakbanken to $dir/corpus_processed. This will take a while."
+
+if [ ! -f $dir/download/sve.16khz.0467-1.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/talegjenkjenning/16kHz/sve.16khz.0467-1.tar.gz --directory-prefix=$dir/download ) &
+fi
+
+if [ ! -f $dir/download/sve.16khz.0467-2.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/talegjenkjenning/16kHz/sve.16khz.0467-2.tar.gz --directory-prefix=$dir/download ) &
+fi
+
+if [ ! -f $dir/download/sve.16khz.0467-3.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/talegjenkjenning/16kHz/sve.16khz.0467-3.tar.gz --directory-prefix=$dir/download ) &
+fi
+
+if [ ! -f $dir/download/sve.16khz.0467-1.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/talegjenkjenning/16kHz/sve.16khz.0468.tar.gz --directory-prefix=$dir/download ) &
+fi    
+wait
+
```

Comment:
a) are you planning to submit the files to openslr? in that case I'd suggest to change the paths right away and add the original mirror as a fallback
b) I think it's not necessary to download it in parallel -- I'm not convined it saves that much time and we generally discourage people to dowload things in parallel from openslr.